web_widget_date_more_filters/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright OpenStudio 2024
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Web date widget more filters",
+    "category": "web",
+    "version": "16.0.1.0.0",
+    "website": "https://github.com/OCA/web",
+    "author": "Elise Gigot <egigot@openstudio.fr>, Odoo Community Association (OCA)",
+    "depends": ["web", "hr_timesheet"],
```

Comment:
Why the dependency on hr_timesheet? Lots of Odoo instances are not using this, so should not be forced to install timesheet, just to have better date selections.

Reply:
Hi @NL66278 ,

Because at the beginning the goal was to replace all the entries by the old ones only for the timesheet and project module, finally we choose to just add them everywhere.

Thank you for your feedback,
We will remove the dependency